satellite/console/apikeys.go

```diff
@@ -47,7 +47,7 @@ type APIKey [24]byte
 
 // String implements Stringer
 func (key APIKey) String() string {
-	return base64.URLEncoding.EncodeToString(key[:])
+	return base32.HexEncoding.EncodeToString(key[:])
```

Comment:
How do people with existing setup convert their keys?

Reply:
From `base64` to `base32`? If yes than `APIKeys` itself is the same, so by converting from `base64` to `[]byte` and then to `base32` string

Reply:
I understand that part. We now have people using the system, so we need to communicate that and exact things that they need to do to upgrade.

Reply:
Something like a small binary, or web page that will handle the conversion?

Reply:
is this change necessary? once we switch to macaroons we're going to have really long keys. if you're trying to avoid `_` and `-` could we perhaps use base58?

@egonelbre i'm okay with api keys becoming invalidated during the alpha from time to time

Reply:
@jtolds Brandon asked me to finish this as soon as possible 
https://storjlabs.atlassian.net/browse/V3-1549